core/renderers/common/path_object.js

```diff
@@ -220,3 +220,19 @@ Blockly.blockRendering.PathObject.prototype.updateInsertionMarker = function(
 Blockly.blockRendering.PathObject.prototype.updateMovable = function(enable) {
   this.setClass_('blocklyDraggable', enable);
 };
+
+/**
+ * Method that's called when the drawer is about to draw the block.
+ * @package
+ */
+Blockly.blockRendering.PathObject.prototype.beginDrawing = function() {
```

Comment:
I don't think you need these on the base path object if you're not putting them onto the interface.  You're only using them in the zelos case, and at that point you know you have a zelos path object.

Reply:
Removed.